planner/implementation/simple_plans.go

```diff
@@ -36,3 +37,20 @@ type ShowImpl struct {
 func NewShowImpl(show *plannercore.PhysicalShow) *ShowImpl {
 	return &ShowImpl{baseImpl: baseImpl{plan: show}}
 }
+
+// SelectionImpl is the implementation of PhysicalSelection.
+type SelectionImpl struct {
+	baseImpl
+}
+
+// CalcCost implements Implementation CalcCost interface.
+func (sel *SelectionImpl) CalcCost(outCount float64, childCosts []float64, children ...*memo.Group) float64 {
+	// TODO: When CPUFactor and CopCPUFactor can be set by users, we need to distinguish them.
```

Comment:
I think we can mark the Group is `BelowGather` or not to distinguish `cop` and `root` task to decide use `CPUFactor` or `CopCPUFactor`. Any idea?

Reply:
Seems we should implement different `Selection` operator for different tasks. For example:
* root task: `RootSelection`
* cop task: `CopSelection`
* flash task: `FlashSelection`